README.md

```diff
@@ -16,6 +16,8 @@ before trusting. For example, `req.body.foo.toString()` may fail in multiple
 ways, for example the `foo` property may not be there or may not be a string,
 and `toString` may not be a function and instead a string or other user input.
 
+**Note** Brotli provides better and faster compression then gzip or deflate, but is supported only since Node.js versions v11.7.0 and v10.16.0.
```

Comment:
Please hard-wrap this line to match the width of the rest of the lines. In addition, since this module only ever decompresses, is this note about compression relevant? I would think that the information about version support should be along with the `br` support docs, otherwise users may not see it.

Reply:
I altered the note

Reply:
Please stop marking this as resolved when you didn't address all my comments or respond back with why you don't think you should address some part to have a discussion on it.

Reply:
I did hard wrap this when resolving the original comment, and now this line does not even exist 😂
This is why it was hidden. It's marked as "outdated" automatically

Reply:
>  I would think that the information about version support should be along with the br support docs, otherwise users may not see it.

Reply:
OMG please STOP
![image](https://user-images.githubusercontent.com/67512/87340606-4bce8700-c516-11ea-95fb-81d81fd54001.png)